### DISCUSSION TITLE
Eleventy not compiling `index.hbs` template inside `integrations` directory

### DISCUSSION THREAD

**Discussion by davegomez** [11ty/eleventy#2886] [Q&A] [answered]
I have the following directory with its own `index.hbs` file:
```
./src/views/integrations/index.hbs
```
I also have other directories with their own `index.hbs` files inside the `integrations` directory, like so:
```
./src/views/integrations/index.hbs
./src/views/integrations/app-one/index.hbs
./src/views/integrations/app-two/index.hbs
```
When compiling the project, Eleventy is not getting the `./src/views/integrations/index.hbs` file, I can't even see it in the list of paths when using the `DEBUG` mode, but if I change the name of the directory to any other like e.g. `integration` it works correctly.

But I can't change the name of the directory, because the URL depends on it, so I don't know what to do to fix this.

**Comment by zachleat** [comment, marked as answer]
Just up front I would note that `permalink` makes file system location decoupled from the URL: https://www.11ty.dev/docs/permalinks/

**Reply by davegomez** [reply]
Yes. I think I might end up using this method for now, but it would be good to know why this might be happening.

Thanks for the swift response BTW

**Comment by pdehaan** [comment]
I was unable to reproduce this with a simple test case: https://github.com/pdehaan/11ty-2886/

```sh
tree src www

src/
└── views/
    └── integrations/
        ├── app-one/
        │   └── index.hbs
        ├── app-two/
        │   └── index.hbs
        └── index.hbs
www/
└── views/
    └── integrations/
        ├── app-one/
        │   └── index.html
        ├── app-two/
        │   └── index.html
        └── index.html

4 directories, 3 files
```
